scale/gcloud_update.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+
+import subprocess
+from settings import GCLOUD_INSTANCE_GROUP, GCE_ZONE
+from utils import getPods, getPodType, getPodNamespace,\
+    get_singleuser_image_value
+
+import logging
+
+
+def shutdownSpecifiedNode(name):
+    """Deletes the specified node by calling the Google Cloud Engine"""
+    cmd = ['gcloud', 'compute', 'instance-groups', 'managed', 'delete-instances',
+           GCLOUD_INSTANCE_GROUP, '--instances=' + name]
+    p = subprocess.Popen(' '.join(cmd), stdout=subprocess.PIPE, shell=True)
+    output, err = p.communicate()
+
+    # TODO: Jeff pls find a proper way to log the below message
+    return "Successfully removed node: %s\n" % name if not err else "Unable to remove node"
```

Comment:
If we can't remove the node, then it is an 'exceptional circumstance' (aka one we don't expect to happen) - so we can just throw an exception, right?

Also in general informational logging like this should be placed where these functions are called, rather than in the functions themselves.